### PR TITLE
Change Organization model and add API

### PR DESCRIPTION
**Pull request by dekkers**
Remove the organization signal_username and signal_group_id fields and some code that references those fields. Not everything related to signal / miauw has been cleaned up yet.

Change Organization code to a lower-case unicode slug field and increase length to 32.

Add tags to Organization models using django-tagulous with a custom tag model with colors and borders.

Add a REST API using django-rest-framework.

### CODE REVIEW

File: .ci/rocky_db_seed.json [hunk 1]
```diff
@@ -20,9 +20,7 @@
     "pk": 1,
     "fields": {
       "name": "test-organization",
-      "code": "_dev",
-      "signal_username": null,
-      "signal_group_id": null
+      "code": "_dev"
```
Comment (underdarknl) [inline .ci/rocky_db_seed.json:23]: Do we still need this _dev in here? I thinik the onboarding flow now creates the first organization.

**Review by underdarknl — CHANGES_REQUESTED** [review]
I really don't wat to include jquery, bootstrap and some other JS libraries for a html interface that is used for the restfull api's.

**Comment by dekkers** [comment]
> I really don't wat to include jquery, bootstrap and some other JS libraries for a html interface that is used for the restfull api's.

I think including the javascript in the package is a lot better than what we currently have with FastAPI that loads the latest version of all the javascript from a third party server (just look at /docs or /redoc on one of our FastAPI services).

I think the browsable API is useful for people who want to use the API, but maybe we should make this a setting and disable the browsable API by default?

**Comment by underdarknl** [comment]
Lets disable the browsable api then. If someone needs it  they can carry the burden of keeping things up to date.
For fastApi api's I feel the same way, although those are not internet connected as easily as rocky is. Minimizing dependencies, especially on the frontend for parts that are hardly used should always be a top priority for a security product like OpenKAT

**Comment by dekkers** [comment]
I have added BROWSABLE_API env var to control whether the browsable API is enabled. It will default to False in production and True when DEBUG is True.

**Review by underdarknl — APPROVED** [review]

**Pull request merged**